azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -130,10 +220,33 @@ public long getLastCleanerThreadCheckTime() {
   }
 
   @Override
+  public Set<Executor> getAllActiveExecutors() {
+    return activeExecutors;
```

Comment:
guess returning a copy of the list might be a better idea as the in memory list is your only  "data of truth",  you may not want the list to be messed up by the caller.